mainmatter/background_and_related_work/short_history/index.tex

```diff
@@ -1,92 +1,137 @@
 \section{The Short History of Graphics \acs{API}s}\label{sec:short_history}
-\begin{sectionmeta}
-	This section gives an overview of the evolution of computer graphics with the focus being on the \glspl{API}.
-	First section will look at graphics before graphics \glspl{API}, the example given will be that of a Commodore 64.
-	After that, the early competition for getting the graphics \gls{API} market is described.
-	Lastly, we look at some of the more modern improvement that is made to the modern \glspl{API}.
-	
-	In this section, the term 3D accelerator is used interchangeably with graphics processor or \gls{GPU}, because this is the historically accurate term. 
-	They all refer to hardware designed to render 3D graphics.
-\end{sectionmeta}
-
-To give perspective to the eventual discussion of modern graphics \glspl{API}, we give a short history of graphics programming and \glspl{API} in this section.
+In this section, we describe the history of graphics programming up to and including the introduction of Direct3D 12 and Vulkan.
+This is done as to place these recent developments in the context of what came before.
+The section will focus on consumer electronics as opposed to hardware used in arcades.
 
 \subsection{Early Days of Consumer Graphics}
+In the early 1980's, dedicated hardware for rendering images became available on home PCs in the form of video display controllers \cite{wikiVideoDisplayController}.
```

Comment:
There is a citation! Use this for when I earlier asked for a citation for this same thing :)